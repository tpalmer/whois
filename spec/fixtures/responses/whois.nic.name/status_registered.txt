Disclaimer: VeriSign, Inc. makes every effort to maintain the
completeness and accuracy of the Whois data, but cannot guarantee
that the results are error-free. Therefore, any data provided
through the Whois service are on an as is basis without any
warranties.
BY USING THE WHOIS SERVICE AND THE DATA CONTAINED 
HEREIN OR IN ANY REPORT GENERATED WITH RESPECT THERETO, IT IS 
ACCEPTED THAT VERISIGN, INC. IS NOT LIABLE FOR 
ANY DAMAGES OF ANY KIND ARISING OUT OF, OR IN CONNECTION WITH, THE 
REPORT OR THE INFORMATION PROVIDED BY THE WHOIS SERVICE, NOR 
OMISSIONS OR MISSING INFORMATION. THE RESULTS OF ANY WHOIS REPORT OR 
INFORMATION PROVIDED BY THE WHOIS SERVICE CANNOT BE RELIED UPON IN 
CONTEMPLATION OF LEGAL PROCEEDINGS WITHOUT FURTHER VERIFICATION, NOR 
DO SUCH RESULTS CONSTITUTE A LEGAL OPINION. Acceptance of the 
results of the Whois constitutes acceptance of these terms, 
conditions and limitations. Whois data may be requested only for 
lawful purposes, in particular, to protect legal rights and 
obligations.  Illegitimate uses of Whois data include, but are not 
limited to, unsolicited email, data mining, direct marketing or any 
other improper purpose.  Any request made for Whois data will be 
documented by VeriSign, Inc. but will not be used for any commercial purpose whatsoever.
 **** 
Domain Name ID: 2788515DOMAIN-NAME
Domain Name: CARLETTI.NAME
Sponsoring Registrar ID: 44REGISTRAR-NAME
Sponsoring Registrar: GoDaddy.com, LLC
Domain Status: clientDeleteProhibited
Domain Status: clientRenewProhibited
Domain Status: clientTransferProhibited
Domain Status: clientUpdateProhibited
Registrant ID: 7903395CONTACT-NAME
Admin ID: 7903397CONTACT-NAME
Tech ID: 7903396CONTACT-NAME
Billing ID: 7903398CONTACT-NAME
Name Server ID: 1016683HOST-NAME
Name Server: NS1.DREAMHOST.COM
Name Server ID: 1016684HOST-NAME
Name Server: NS2.DREAMHOST.COM
Created On: 2006-04-19T12:22:08Z
Expires On: 2013-04-19T12:22:08Z
Updated On: 2011-10-17T18:42:56Z

