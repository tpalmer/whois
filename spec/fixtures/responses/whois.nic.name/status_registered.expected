#status
  should: %s == ["clientDeleteProhibited", "clientRenewProhibited", "clientTransferProhibited", "clientUpdateProhibited"]

#available?
  should: %s == false

#registered?
  should: %s == true


#created_on
  should: %s CLASS(time)
  should: %s == Time.parse("2006-04-19 12:22:08 UTC")

#updated_on
  should: %s CLASS(time)
  should: %s == Time.parse("2011-10-17 18:42:56 UTC")

#expires_on
  should: %s CLASS(time)
  should: %s == Time.parse("2013-04-19 12:22:08 UTC")


#nameservers
  should: %s CLASS(array)
  should: %s SIZE(2)
  should: %s[0] CLASS(nameserver)
  should: %s[0].name == "ns1.dreamhost.com"
  should: %s[1] CLASS(nameserver)
  should: %s[1].name == "ns2.dreamhost.com"
